src/prefect/cli/kubernetes.py

```diff
@@ -24,9 +24,14 @@
 )
 app.add_typer(kubernetes_app)
 
+# Deprecate kubernetes maniest
 manifest_app = PrefectTyper(
     name="manifest",
-    help="Commands for generating Kubernetes manifests.",
+    help="Deprecated. Use the 'Helm charts' instead.",
```

Comment:
I think that we can retain the previous help as this should be covered by the `deprecated_help`. Maybe 
```suggestion
    help="Commands for generating Kubernetes manifests. Deprecated.",
```